notebook/services/contents/filemanager.py

```diff
@@ -284,7 +286,9 @@ def _dir_model(self, path, content=True):
                         self.log.warning("Error stat-ing %s: %s", os_path, e)
                     continue
 
-                if not stat.S_ISREG(st.st_mode) and not stat.S_ISDIR(st.st_mode):
+                if (not WINDOWS_CONTAINER
```

Comment:
As it stands the `WINDOWS_CONTAINER` flag both tests that the platform is `win32` *and* that you're running from inside a container by checking for the default container user `ContainerAdministrator`.

It's certainly possible to make the changes for all windows users but I was hesitant to potentially break existing code.